script/rust_deps_config.py

```diff
@@ -8,4 +8,4 @@
 # it now breaks because of urllib2 and SSL/TLS incompatibility when using Fastly
 # RUST_DEPS_PACKAGES_URL = "https://rust-pkg-brave-core.s3.brave.com"
 RUST_DEPS_PACKAGES_URL = "https://rust-pkg-brave-core.s3.amazonaws.com"
-RUST_DEPS_PACKAGE_VERSION = "0.1.2"
+RUST_DEPS_PACKAGE_VERSION = "0.1.3"
```

Comment:
I assume that `https://rust-pkg-brave-core.s3.amazonaws.com @ 0.1.3` contains an updated version of stable `rustc` (e.g. 1.41^^). If that's the case, LGMT (note: I didn't build the browser with these changes)

Reply:
0.1.3 contains latest rustc